src/status_im2/contexts/onboarding/create_profile/view.cljs

```diff
@@ -129,15 +130,21 @@
                                                          :else           :success)
           {:keys [keyboard-shown keyboard-height]} (hooks/use-keyboard)
           show-background?                         (show-button-background keyboard-height
-                                                                           keyboard-shown)]
+                                                                           keyboard-shown
+                                                                           @content-scroll-y)]
       [rn/view {:style style/page-container}
        [navigation-bar/navigation-bar
         {:stack-id :new-to-status
          :top      navigation-bar-top}]
        [rn/scroll-view
         {:on-layout               (fn [event]
                                     (let [height (oops/oget event "nativeEvent.layout.height")]
-                                      (reset! scroll-view-height height)))
+                                      (reset! scroll-view-height height)
+                                      (reset! content-scroll-y 0)))
+         :on-scroll               (fn [event]
+                                    (let [y (oops/oget event "nativeEvent.contentOffset.y")]
+                                      (reset! content-scroll-y y)))
+         :scrollEventThrottle     64
```

Comment:
can be kebab-case? we use 64 elsewhere in the codebase? perhaps this should be a constant somewhere?

Reply:
we just use 64 :( but also it's different for different lists depending on refresh rate we expect